crates/symbolicator/src/endpoints/symbolicate_jvm.rs

```diff
@@ -64,13 +64,17 @@ pub async fn handle_symbolication_request(
         platform,
         sources,
         exceptions,
-        stacktraces,
+        mut stacktraces,
         modules,
         release_package,
         classes,
         options,
     } = body;
 
+    // Sentry sends stacktraces in "Sentry order" (innermost frame at the end). We want them
+    // in "Symbolicator order" (innermost frame at the front).
+    stacktraces.iter_mut().for_each(|st| st.frames.reverse());
```

Comment:
Feel free to leave as is, but I personally follow this clippy lint and try to avoid `iterator.for_each`: https://rust-lang.github.io/rust-clippy/master/index.html?search=for_each#needless_for_each

Reply:
Yeah fair enough, not sure why I wanted `for_each` here to begin with.